tests/phpunit/WC_Stripe_Payment_Gateway_Test.php

```
@@ -844,4 +845,108 @@ public function test_invalid_request_error_with_non_blocked_code_is_retryable()
+				'is subscription'  => true,
+				'payment method'   => [
+					'id'                     => 'pm_789',
+					'type' => 'card',
+					'card' => [
+						'country' => 'IN',
+					],
+				],
+				'expected'         => false,
+			],
+		];
+	}
```

PHPMD (2.15.0)

[error] The method provide_test_disable_subscription_edit_for_india is not named in camelCase. (undefined)

(CamelCaseMethodName)